test/core/consensus/synchronizer/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+addtest(synchronizer_test
+    synchronizer_test.cpp
+    )
+target_link_libraries(synchronizer_test
+    synchronizer
+    buffer
```

Comment:
should not be here. Instead link synchronizer with buffer (synchronizer uses BlockData which has buffer)